Makefile

```diff
@@ -465,13 +468,6 @@ verify-gen: generate ## Verfiy go generated files are up to date
 		echo "generated files are out of date, run make generate"; exit 1; \
 	fi
 
-.PHONY: verify-manifests
-verify-manifests: manifests ## Verfiy go generated manifests files are up to date
```

Comment:
@mkumatag we should preserve this one with `generate-manifests` target right?

Reply:
This will get tested via `verify-generate` -> `generate` -> `generate-manifests`